apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/PartitionOptions.cs

```diff
@@ -12,25 +12,11 @@
 // See the License for the specific language governing permissions and 
 // limitations under the License.
 
-using System.Threading;
-
 namespace Google.Cloud.Spanner.Data;
 
 /// <summary>
-/// An immutable class representing partition options.
+/// Options used to create and read partitions.
```

Comment:
I think leaving something in here about immutability is useful, as it indicates that users can reuse options instead of recreating them.